RecoTracker/LST/plugins/alpaka/LSTProducer.cc

```diff
@@ -19,6 +18,10 @@
 #include "RecoTracker/LST/interface/LSTPhase2OTHitsInput.h"
 #include "RecoTracker/LST/interface/LSTOutput.h"
 
+#ifdef ALPAKA_ACC_GPU_CUDA_ENABLED
```

Comment:
Maybe remove these `ifdef`, given that (for now) we specify `<flags ALPAKA_BACKENDS="cuda"/>` in the BuildFile?